controllers/deployment.go

```diff
@@ -40,15 +40,18 @@ type NetworkServiceName = string
 func (d *DeploymentPlan) Init() {
 	d.planMap = make(map[int32]ServiceGroups)
 
-	// Default: each service runs in its own Pod
+	// Default: each service group runs in its own Pod and each service in it's own container.
+	// If there are mltiple services in a pod,
+	// the first one (index 0) will be considered as the
+	// main container and subsequent ones as sidecar containers.
 	d.planMap[0] = ServiceGroups{
 		stateful: map[ServiceGroupName]ServiceGroup{
```

Comment:
May be use a ServiceGroupInfo struct here that has the ServiceGroup and a boolean for collecting metrics.

Reply:
Acknowledged